modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -37,6 +38,52 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.11.0
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.11 that need your attention. You can also read {oc-changelog-url}[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+When upgrading from ownCloud Server 10.10, there will be no migrations steps impacting the upgrade duration. Therefore the upgrade is expected to finish quickly.
+
+=== Edit Permission for Public Links on Single Files
+
+In previous versions of ownCloud Server, public links on single files could only be created with read-only permissions (`Download / View`). If users wanted to share a file and enable recipients to change it, they had to put that file into a folder and share that with permissions for editing (`Download / View / Edit`). Server 10.11 introduces the `Download / View / Edit` role for public link shares on single files to make a user's life easier. Especially when using ownCloud in combination with web office solutions like ONLYOFFICE, Collabora Online or Microsoft Office Online, users can now quickly and conveniently collaborate on documents with external parties. https://github.com/owncloud/core/pull/40264[#40264]
+
+=== Sharing with Multiple Users at once
+
+Server 10.11 introduces the ability to share with multiple users/groups at once by entering their identifiers (e.g., display name, user ID, email address, federated cloud ID) as a comma-separated list in the sharing dialog (e.g., `alice,bob@example.org,carol,teachers`. Based on the inputs, ownCloud will decide how to create the shares (e.g., share with an internal user, invite a new guest, create a federated share). This way, users can easily invite many recipients at once to their files and folders. With Custom Groups version 0.7.0 onwards the same behavior also applies to adding users to Custom Groups. https://github.com/owncloud/core/pull/40155[#40155] https://github.com/owncloud/customgroups/pull/522[customgroups#522] https://github.com/owncloud/guests/pull/506[guests#506]
```

Comment:
Need to check this - I thought that the sharing UI on the classic webUI expects the comma-separated list to be a list of users. Groups can't be included in the list. (I remember testing with a group called "brian,carol" and 2 users "brian" and "carol" to check what happens if there is also a group name that happens to be exactly the comma-separated list of user names)

So I think that the example can't have a group called "teachers" at the end.

Reply:
You're right. Will remove that. Thanks for checking carefully.